### PR TITLE
plugins:foreman_docker - 2.1.1 [rpm]

### PR DESCRIPTION
**Pull request by dLobatog**
For Foreman 1.11+

### CODE REVIEW

File: rubygem-foreman_docker/rubygem-foreman_docker.spec [hunk 1]
```diff
@@ -17,23 +17,27 @@
...
+Version:    2.1.0
 Release:    1%{?foremandist}%{?dist}
 Group:      Applications/System
 License:    GPLv3
 URL:        http://github.com/theforeman/foreman-docker
 Source0:    http://rubygems.org/downloads/%{gem_name}-%{version}.gem
 
 Requires:   foreman-compute >= 1.11.0
-Requires:   %{?scl_prefix}rubygem(docker-api) = 1.17.0
+Requires:   %{?scl_prefix}rubygem(docker-api) >= 1.18.0
```
Comment (domcleal) [inline rubygem-foreman_docker/rubygem-foreman_docker.spec]: You will need to update this package too.

**Comment by dLobatog** [comment]
This needs an updated docker-api gem, I'll submit a PR shortly

File: rubygem-foreman_docker/rubygem-foreman_docker.spec [hunk 2]
```diff
@@ -17,23 +17,27 @@
...
+Requires:   %{?scl_prefix}rubygem(docker-api) < 2.0
 Requires:   %{?scl_prefix}rubygem(wicked) >= 1.1
-Requires:   %{?scl_prefix}rubygem(wicked) < 2.0
+Requires:   %{?scl_prefix}rubygem(wicked) >= 1.1
```
Comment (domcleal) [inline rubygem-foreman_docker/rubygem-foreman_docker.spec]: This duplicates the line above?

File: rubygem-foreman_docker/rubygem-foreman_docker.spec [hunk 3]
```diff
@@ -17,23 +17,27 @@
...
 License:    GPLv3
 URL:        http://github.com/theforeman/foreman-docker
 Source0:    http://rubygems.org/downloads/%{gem_name}-%{version}.gem
 
 Requires:   foreman-compute >= 1.11.0
-Requires:   %{?scl_prefix}rubygem(docker-api) = 1.17.0
+Requires:   %{?scl_prefix}rubygem(docker-api) >= 1.18.0
+Requires:   %{?scl_prefix}rubygem(docker-api) < 2.0
 Requires:   %{?scl_prefix}rubygem(wicked) >= 1.1
-Requires:   %{?scl_prefix}rubygem(wicked) < 2.0
```
Comment (domcleal) [inline rubygem-foreman_docker/rubygem-foreman_docker.spec]: This is still in the gemspec.

File: rubygem-foreman_docker/rubygem-foreman_docker.spec (hunk 1 above)
Comment (dLobatog) [inline rubygem-foreman_docker/rubygem-foreman_docker.spec]: This literally showed up when I clicked on "Comment" :smile:

**Comment by domcleal** [comment]
> For Foreman 1.11+

I notice the gem now uses `new_link`, which IIRC was only introduced in Foreman 1.12.  Are you sure it's compatible with 1.11 (have tests run on it)?

Also, the git repo's missing a 2.1.0 tag.

**Comment by dLobatog** [comment]
`undefined method `new_link'` - my bad - I guess I'll have to change this and do 2.1.1

**Comment by dLobatog** [comment]
https://github.com/theforeman/foreman-docker/pull/159 - when I merge it, I'll release 2.1.1 and we'll be able to move forward with ths PR.

**Comment by dLobatog** [comment]
Aight, `No Package found for tfm-rubygem(docker-api) >= 1.18.0`  - pending https://github.com/theforeman/foreman-packaging/pull/1094

**Comment by domcleal** [comment]
[test]

**Comment by domcleal** [comment]
Merged and will build, thanks @dLobatog!

**Pull request merged**